test/lbp.jl

```diff
@@ -245,18 +245,22 @@ facts("Local Binary Patterns") do
 		@fact multi_block_lbp(img, 1, 1, 4, 4) --> 18
 	end
 
-	context("Descriptor") do
+	#=context("Descriptor") do
```

Comment:
Yeah. You can take a simple binary image (1s and 0s) say of size 10x10. For a block size of 5x5, we will have 4 histograms which will be created from the LBP images of each 5x5 image.